resources/public/api/conf/1.0/examples/checkpoints/list_response.json

```diff
@@ -0,0 +1,48 @@
+{
+  "checkpoints": [
+    {
+      "checkpointId": "a1c8057e-fbbc-47a8-a8b4-78d9f015c253",
+      "nino": "TC663795B",
+      "checkpointCreationTimestamp": "2019-07-13T07:51:43.112Z",
+      "links": [
+        {
+          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints/a1c8057e-fbbc-47a8-a8b4-78d9f015c253/restore",
+          "method": "POST",
+          "rel": "restore-checkpoint"
+        },
+        {
+          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints/a1c8057e-fbbc-47a8-a8b4-78d9f015c253",
+          "method": "DELETE",
+          "rel": "delete-checkpoint"
+        },
+        {
+          "href": "/individuals/self-assessment-test-support/vendor-state/checkpoints?nino={nino}",
+          "method": "POST",
+          "rel": "create-checkpoint"
+        }
+      ]
+    },
+    {
+      "checkpointId": "b2e4050e-fbbc-47a8-d5b4-65d9f015c253",
+      "nino": "RC972995B",
+      "checkpointCreationTimestamp": "2019-07-13T08:51:43.112Z",
```

Comment:
Shall we make this timestamp different to the timestamp on line 6? Would make the example more realistic imo

Reply:
I had changed the time but not the date - will update it to have a different date now

Reply:
Whoops didn't see the difference in the hour! Thinking about it now, it probably makes more sense to put the creationTimestamps back to how you had it before - since checkpoints will be automatically deleted after 7 days, the current example (with one checkpoint created in 2019 and another in 2020) is impossible. Soz!